### PR TITLE
Added examples to read me documentation

### PR DESCRIPTION
**Pull request by mtpjr88**
I use the library at my company and added some very useful examples that I think others can find useful.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -74,7 +74,38 @@ hubspot.contacts.get(function(err, results) {
...
+                { "property": "city", "value": yourvalue }
+            ]
+        };
+
+          const hubspot = new Hubspot({ apiKey: YOUR API KEY });
+          const hubspotContact = await hubspot.contacts.create(contactObj);
+```
+## {EXAMPLE} If you need to insert mulitple values. each value must have a semi colon after each value
+```javascript
+const arrayOfValues = ["value1", "value2", "value3", "value4"];
```
Comment (pcothenet) [inline README.md]: I'm not sure I understand this. Do you need to use this for a specific HubSpot data type or is it a general explanation (to cast an array into a string value)?

**Review by pcothenet — CHANGES_REQUESTED** [review]

**Comment by pcothenet** [comment]
Thanks @mtpjr88. The examples are definitely useful but I would revise the proposal a bit to make it lighter (and specific to node-hubspot)

File: README.md (hunk 1 above)
Comment (mtpjr88) [reply README.md]: Yes if you send hubspot multiple items it must be delimited by a semi colon.

**Comment by mtpjr88** [comment]
I showed examples of exactly how hubspot expects your payloads. Do you have specifics so I can revise it?

**Comment by mtpjr88** [comment]
would you be able to merge with the changes I have?

**Review by pcothenet — APPROVED** [review]

**Comment by mtpjr88** [comment]
awesome thank you!

**Pull request merged**